src/traces/heatmap/calc.js

```diff
@@ -211,14 +218,17 @@ function makeBoundArray(trace, arrayIn, v0In, dvIn, numbricks, ax) {
     }
     else {
         dv = dvIn || 1;
-        if(v0In === undefined) v0 = 0;
+
+        if(Array.isArray(arrayIn) && arrayIn.length === 1) v0 = arrayIn[0];
+        else if(v0In === undefined) v0 = v0In;
```

Comment:
Good :eyes: should be `if(v0In === undefined) v0 = 0;` like the line I removed. Thanks!